src/main/java/app/bpartners/api/service/UserService.java

```diff
@@ -116,4 +119,22 @@ public void deleteUserByEmail(String email) {
     userRepository.deleteById(user.getId());
     cognitoComponent.deleteUserByUsername(email);
   }
+
+  @Transactional
+  public void registerOnStripeActiveUsersWithNullSubscription() {
+    List<User> users = userRepository.getActiveUsersWithNullSubscription();
+    var totalUser = users.size();
+    int[] userNb = {0};
+    users.forEach(
+        user -> {
+          UserNonSubscribed event =
+              UserNonSubscribed.builder()
+                  .userId(user.getId())
+                  .totalNbUser(totalUser)
+                  .userNb(userNb[0])
```

Comment:
What is the use of totalNbUser and userNb here ?

Reply:
TotalNb and UserNB are used in the mail. In the mail, we specify the number of users out of how many have been registered. TotalNb refers to the total number of users who are active and not yet registered, while UserNb refers to the number of users registered during the execution of the request.